docs/source/checks/nlp/model_evaluation/plot_confusion_matrix_report.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please check also output (you added one of a the Single Dataset Performance and not of the Confusion matrix)